binaries/cuprated/src/blockchain/syncer.rs

```diff
@@ -1 +1,139 @@
+// FIXME: This whole module is not great and should be rewritten when the PeerSet is made.
```

Comment:
I have made a syncer - it works but I have plans to improve it with some P2P changes